HappyAnding/HappyAnding/ViewModel/CheckUpdateVersion.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  CheckUpdateVersion.swift
+//  HappyAnding
+//
+//  Created by 전지민 on 2022/11/29.
+//
+
+import UIKit
+import SwiftUI
+import Combine
+import FirebaseFirestore
+
+struct Version: Codable {
+    var latestVersion: String
+    var minimumVersion: String
+    var description: String
+    var title: String
+    
+    enum CodingKeys: String, CodingKey {
+        case latestVersion = "latest_version"
+        case minimumVersion = "minimum_version"
+        case title, description
+    }
+}
+
+class CheckUpdateVersion {
+    
+    static let share = CheckUpdateVersion()
+    private let db = Firestore.firestore()
+    
+    func fetchVersion(completionHandler: @escaping (Version, Bool)->()) {
+        
+        db.collection("Version").addSnapshotListener { snapshot, error in
+            guard let snapshot else {
+                print("Error fetching snapshots: \(error!)")
+                return
+            }
+            snapshot.documentChanges.forEach { diff in
+                let decoder = JSONDecoder()
+                
+                do {
+                    let data = diff.document.data()
+                    let jsonData = try JSONSerialization.data(withJSONObject: data)
+                    let version = try decoder.decode(Version.self, from: jsonData)
+                    
+                    let localVersionString = Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as! String
+                    let isNeeded = self.checkIsNeededForceUpdate(localVersion: localVersionString, minimumVersion: version.minimumVersion)
+                    
+                    completionHandler(version, isNeeded)
+                    
+                } catch let error {
+                    print("error: \(error)")
+                }
+            }
+        }
+    }
+
+    
+    func checkIsNeededForceUpdate(localVersion: String, minimumVersion: String) -> Bool {
+        
+        if (self.compareVersion(versionA: localVersion, versionB: minimumVersion) == ComparisonResult.orderedAscending) {
+            return true
+        } else {
+            return false
+        }
+    }
```

Comment:
```suggestion
    func checkIsNeededForceUpdate(localVersion: String, minimumVersion: String) -> Bool {
        self.compareVersion(versionA: localVersion, versionB: minimumVersion) == ComparisonResult.orderedAscending
    }
```

조건 자체를 return 시킬 수 있을 것 같네요 ◡̈
+ 라인 한 줄이라 return 키워드도 생략 가능하겠군요!